fury-on-ovhcloud/README.md

```diff
@@ -0,0 +1,539 @@
+  - name: logging
+  - name: ingress
+```
+
+### Download Fury modules
+
+1. Download the Fury modules with `furyctl`:
+
+```bash
+furyctl vendor -H
+```
+
+2. Inspect the downloaded modules in the `vendor` folder:
+
+```bash
+tree -d vendor -L 3
+```
+
+Output:
+
+```bash
+$ tree -d vendor -L 3
+
+vendor
+└── katalog
+    ├── ingress
+    │   ├── cert-manager
+    │   ├── dual-nginx
+    │   ├── external-dns
+    │   ├── forecastle
+    │   ├── nginx
+    │   └── tests
+    ├── logging
+    │   ├── cerebro
+    │   ├── configs
+    │   ├── logging-operated
+    │   ├── logging-operator
+    │   ├── loki-configs
+    │   ├── loki-single
+    │   ├── opensearch-dashboards
+    │   ├── opensearch-single
+    │   ├── opensearch-triple
+    │   └── tests
+    └── monitoring
+        ├── aks-sm
+        ├── alertmanager-operated
+        ├── blackbox-exporter
+        ├── configs
+        ├── eks-sm
+        ├── gke-sm
+        ├── grafana
+        ├── kubeadm-sm
+        ├── kube-proxy-metrics
+        ├── kube-state-metrics
+        ├── node-exporter
+        ├── prometheus-adapter
+        ├── prometheus-operated
+        ├── prometheus-operator
+        ├── tests
+        ├── thanos
+        └── x509-exporter
+```
+
+## Step 3 - Installation
+
+We use Terraform to deploy the network and the Managed Kubernetes Cluster. A `variables.tfvars` variable file is present with some default values that you can use as this or change the values if needed:
+
+```terraform
+# Region
+  
+region = "GRA7"
+
+# Network - Private Network
+
+network = {
+  name = "furyNetwork"
+}
+
+# Network - Subnet
+
+subnet = {
+  name       = "furySubnet"
+  cidr       = "10.0.0.0/24"
+  dhcp_start = "10.0.0.100"
+  dhcp_end   = "10.0.0.254"
+}
+
+# Network - Router
+
+router = {
+  name = "furyRouter"
+}
+
+# Managed Kubernetes Cluster
+
+kube = {
+  name            = "furykubernetesCluster"
+  pv_network_name = "furyNetwork"
+  gateway_ip      = "10.0.0.1"
+}
+
+pool = {
+  name          = "furypool"
+  flavor        = "b2-7"
+  desired_nodes = "3"
+  max_nodes     = "6"
+  min_nodes     = "3"
+}
+```
+
+- `region`: The main region where you want to deploy your infrastructure.
+
+- `network`: The network name.
+
+- `subnet`: The subnet parameters, like the CIDR IP format value and DHCP range.
+
+- `router`: The router name.
+
+- `kube`: The Managed Kubernetes Cluster, such as its name and essentially network informations.
+
+- `pool`: The Kubernetes node pool parameters.
+
+Deploy the infrastructure with:
+
+```bash
+cd infrastructure/terraform
+
+terraform init
+terraform plan -var-file=variables.tfvars
+terraform apply -var-file=variables.tfvars
+```
+
+Wait a few minutes until the end of the deployement.
+
+### Configure your kubectl environment
+
+Once the Managed Kubernetes Cluster created, in the `terraform` folder, you get the associated `kubeconfig` file. Set the KUBECONFIG variable value like this:
+
+```bash
+export KUBECONFIG="$(pwd)/kubeconfig"
```

Comment:
better to use $PWD (an env var), than catching the output of a subshell with `$()`:
```suggestion
export KUBECONFIG="$PWD/kubeconfig"
```

Reply:
Done (thx)